.circleci/config.yml

```diff
@@ -0,0 +1,146 @@
+
+version: 2
+jobs:
+  build:
+    docker:
+      # openjdk provides Scala dependencides. We may want to consider rolling
+      # our own image to also include JavaScript dependencies. Would speed up CI
+      # at the cost of maintaining a Docker image.
+      - image: openjdk:8
+        environment:
+          CI_TERRIBLENESS: 30.seconds
+        # Run two containers, $CIRCLE_NODE_TOTAL = 2, $CIRCLE_NODE_INDEX = [0|1]
+        parallelism: 2
+    working_directory: ~/linkerd
+    # All run steps execute in both containers unless otherwise specified.
+    # Deploy steps execute on container 0.
+    steps:
+      # Check out linkerd source code into ~/linkerd.
+      - checkout
+
+      # This allows us to run docker commands Since we're already in a docker
+      # container, this step sets up a remote docker environment for us.
+      - setup_remote_docker
+
+      # cat the entire contents of the project/ directory, to be used in cache
+      # keys.
+      - run:
+          name: Generate cache seeds
+          command: |
+            set -x
+            find ~/linkerd/project/ -type f -exec cat {} \; > project-seed.txt
```

Comment:
might as well append admin/src/main/resources/io/buoyant/admin/package.json into this as well.  also i might be inclined to put this in /tmp